youtube-gpt-client/src/components/header/index.tsx

```diff
@@ -1,27 +1,101 @@
-import { Flex, Heading } from '@chakra-ui/react';
+import { Button, Flex, Heading, IconButton, Switch } from '@chakra-ui/react';
 import { useTranslation } from 'react-i18next';
 import LanguageSwitcher from '@/components/header/languageSwitcher';
 import Link from 'next/link';
+import { HamburgerIcon, CloseIcon } from '@chakra-ui/icons';
+import { useState } from 'react';
 
 const Header = () => {
   const { t } = useTranslation('menu');
+  const [display, changeDisplay] = useState('none');
   return (
-    <Flex
-      justifyContent="space-between"
-      alignItems="center"
-      padding="15px 50px"
-      color="#fff"
-      height="60px"
-    >
-      <Heading size="md">
-        <Link href="/">Youtube GPT</Link>
-      </Heading>
-      <Flex gap="20px" fontSize="20px">
-        <Link href="/how-it-works">{t('howItWorks')}</Link>
-        <Link href="/team">{t('team')}</Link>
-        <LanguageSwitcher />
+    <>
+      <Flex
+        justifyContent="space-between"
+        alignItems="center"
+        padding="15px 50px"
+        color="#fff"
+        height="60px"
+        display={['none', 'none', 'flex', 'flex']}
```

Comment:
Don't get what is this line for. The Chakra-UI Flex component has already `display: flex` as default.

Reply:
I don't want to show this component on mobile devices, so we are using the [responsive styles](https://chakra-ui.com/docs/styled-system/responsive-styles) to hide it